src/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, Redux } from 'react-redux';
+import bookReducer from './redux/books/books';
+import statusReducer from './redux/categories/categories';
+
+const rootReducer = combineReducers({
+  books: bookReducer,
+  status: statusReducer,
+});
+
+const store = Redux.createStore(rootReducer);
+
```

Comment:
Kindly refactor the way you create your store as well. We do not need the `Redux` component since it does not exist.
```
  const store = createStore(rootReducer);
```